### ISSUE TITLE
(Idea) Make xfdashboard toggle on screen corner

### ISSUE THREAD

**Issue by alejandro-perez** [gmc-holle/xfdashboard#94]
So far, I have tried to use "brightside", but it seems overkill, as it depends on a lot of gnome packages. How difficult would it make xfdashboard (in daemon mode) to watch mouse position and determine whether the mouse has been on (0,0) for 100ms, then toggle?

If you are not insterested, could you give me some hints on where should I touch the code to try to implement this myself? (e.g. file, kind of signal...)

**Comment by gmc-holle** [comment]
It's nearly the same problem as described in issue #69 for hotkey implementation. Now its a mouse logger and not a key logger but it doesn't make it better.

If "brightside" is overkill what about "xdotool"?

**Comment by alejandro-perez** [comment]
xdotool makes the trick perfectly. Thanks!

**Comment by alejandro-perez** [comment]
Just to make a note, xdotool did not make the trick. For some reason, it has some funny behaviour and when it is running, the pointer never changes (i.e. I don't get the hand icon when hovering a web link, or the resize icon...). 

So I did my own mouse monitoring app, which in turn was extremely easy to do. I took the idea from brightside, but I used python to do it. Less than 40 lines :).

If you want to mention anywhere, it upload it to bitbucket. 
https://bitbucket.org/aperezmendez/corner-watcher

Regards
